package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cucumber-junit-convert",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

```suggestion
  "version": "2.2.1",
```
Package is now on version 2.2.0 for some reason